### PR TITLE
Adds Artifacts for Landscaper Integration

### PR DESCRIPTION
**Pull request by jschicktanz**
**What this PR does / why we need it**:
Adds artifacts for Landscaper integration. Pipeline now produces a CTF that contains the Component Descriptor, Blueprint, and Helm Chart (as Blobs) for this component.

Co-authored by @enrico-kaack-comp 

**Which issue(s) this PR fixes**:
https://github.com/gardener/landscapercli/issues/64

**Special notes for your reviewer**:

**Release note**:

```other operator

```

### CODE REVIEW

**Comment by jschicktanz** [comment]
@schrodit

File: .landscaper/blueprint/deploy-executions.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+          - kind: DNSProvider
+            type: alicloud-dns
+          - kind: DNSProvider
+            type: azure-dns
+          - kind: DNSProvider
+            type: google-clouddns
+          - kind: DNSProvider
+            type: openstack-designate
+          - kind: DNSProvider
+            type: cloudflare-dns
```
Comment (MartinWeindel) [inline .landscaper/blueprint/deploy-executions.yaml:31]:
```suggestion
            type: cloudflare-dns
          - kind: DNSProvider
            type: infoblox-dns
          - kind: DNSProvider
            type: netlify-dns            
```

Comment (MartinWeindel) [reply .landscaper/blueprint/deploy-executions.yaml:31]: add missing DNS provider types

File: .landscaper/resources.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+---
+type: blueprint
+name: dns-controller-manager-blueprint
```
Comment (MartinWeindel) [inline .landscaper/resources.yaml:3]:
```suggestion
name: dns-controller-manager-controller-registration
```

**Review by MartinWeindel — COMMENTED** [review]
Uwe and I have some suggestions

**Comment by jschicktanz** [comment]
Hi, thank you for your review. We have included your feedback and additionally made 1 small change which makes the chart values in the blueprint optional (for enabling the installation with just the default values).

File: .landscaper/resources.yaml [hunk 3]
```diff
@@ -0,0 +1,18 @@
+---
+type: landscaper.gardener.cloud/blueprint
+name: dns-controller-manager-controller-registration
+relation: local
+input:
+  type: "dir"
+  path: "./blueprint"
+  compress: true
+  mediaType: "application/vnd.gardener.landscaper.blueprint.v1+tar+gzip"
```
Comment (schrodit) [inline .landscaper/resources.yaml:9]:
```suggestion
  mediaType: "application/vnd.gardener.landscaper.blueprint.v1.tar+gzip"
```

File: .landscaper/resources.yaml [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+---
+type: helm.io/chart
+name: dns-controller-manager-chart
+relation: local
+input:
+  type: "dir"
+  path: "../charts/external-dns-management"
+  compress: true
```
Comment (schrodit) [inline .landscaper/resources.yaml:17]: add preserveDir

**Review by schrodit — CHANGES_REQUESTED** [review]

**Review by schrodit — APPROVED** [review]

**Pull request merged**